lib/AliTV.pm

```diff
@@ -268,9 +268,9 @@ sub get_json
                                     },
                          'links' => {
                                       'invisibleLinks' => {},
-                                      'maxLinkIdentity' => $self->{_links_max_id}+0,
+                                      'maxLinkIdentity' => ($self->{_links_max_id} >= 100) ? 100: int($self->{_links_max_id}+1),
```

Comment:
This works as intended but is not easy to understand at first glance. Especially the int($self...+1) part. It might be more readable to use `ceil` from the `POSIX` package. Not sure if this is a core module or if it introduces another dependency. In this case I'll vote for keeping your implementation